src/SDKs/DevTestLabs/DevTestLabs.Tests/Tests/BasicTests.cs

```diff
@@ -20,7 +20,7 @@ public void ListLabsTest()
             {
                 var client = GetDevTestLabsClient(context);
 
-                var labs = client.Lab.ListBySubscription().ToList();
+                var labs = client.Labs.ListBySubscription().ToList();
```

Comment:
@ryanvog I am sure I am overlooking something, but please explain me, why there isn't test coverage for a breaking change?